x-pack/plugins/observability_ai_assistant/public/utils/get_timeline_items_from_conversation.tsx

```diff
@@ -88,15 +84,13 @@ export function getTimelineItemsfromConversation({
               arguments: JSON.parse(functionCall.arguments || '{}'),
             });
 
-            const fn = functions.find((func) => func.options.name === message.message.name);
-
-            element = fn?.render ? (
+            element = (
```

Comment:
How do you test if this element actually returns something?

Reply:
There's no difference from what it was before - the RenderFunction component returns the rendered element, or `null`, which has the same end result as before. It's just encapsulated in the RenderFunction now.

Reply:
Actually, this isn't true, there's no difference in rendering, but there is a difference in the value of `element`. Let me check the ramifications.

Reply:
fixed in https://github.com/dgieselaar/kibana/pull/38/commits